charts/beyla/templates/daemon-set.yaml

```diff
@@ -25,9 +25,6 @@ spec:
         {{- with .Values.podAnnotations }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
-        {{- if not (.Values.privileged) }}
-        container.apparmor.security.beta.kubernetes.io/beyla: "unconfined"
```

Comment:
Actually we set this when the container is NOT privileged, so we can add the required capabilities. I don't remember if this was required only for SYS_ADMIN capability, or setting the rest of capabilities require it too.

Reply:
From what I remember we had to use this "unconfined" because App Armour prevents us from mounting file systems or writing to /sys/fs. So if we don't have to mount anymore we don't need to remove this restriction.

However we shouldn't remove this in this PR. It should be a follow-up and we should probably confirm this is  works now with real k8s install, since we have no tests for this.